Peer.Domain/Commands/OpenArguments.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Peer.Domain.Commands
+{
+    public class OpenArguments
+    {
+        public string Partial { get; }
```

Comment:
`Partial` could be a type so we could catch invalidity before we compare against actual PRs. Doesn't make much difference here where the `open` command isn't going to get much more complicated; especially since it's the `Identifier` type that compares the string against itself.

Reply:
That's a good idea. I think it's something good to generalize since we're going to want it for describe/details as well